packages/svg-sprites/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@fluentui/svg-icon-sprites",
```

Comment:
Probably rename the folder for consistency.

Reply:
Maybe `"@fluentui/react-svg-icon-sprites"`

Reply:
'fluent' seems redundant since it's already scoped to @fluentui, and the adjacent package is named `@fluentui/svg-icons`.

Maybe `@fluentui/svg-sprites`? I'll go with that name and adjust the directory name for now, let me know if it should change.

Reply:
Ah, and that happens to be the name of the directory already.

Reply:
I was suggesting the package and directory names should both be `svg-icon-sprites`. By itself I don't think `svg-sprites` alone sufficiently identifies them as icons. It could be any svg content, but I think the consistency in naming between all modern `icon` packages is beneficial.